### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_Databasebackup-networkshare_Keyboard:There is no visible indication of mark provided for password field that it is a required field.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15766]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG3.3.1;#AINotInScope;#Keyboard

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till Migrate to azure sql and press enter.
5. Add azure account and tab till next button and press enter.
6. Select SQL server on azure virtual machine radio button and fill all the required fields and tab till next button and press enter.
7. Check Online migration radio button and tab till next button and press enter.
8. Check "My database are on a network share" radio button.
9. Tab till Password textbox and check if there is a visible indication of asterix mark provided for the field.

### Actual:
There is no visible indication of mark provided for password field that it is a required field.
This issue is observed with all the required fields.

### Expected:
There should be a asterix mark provided for the required fields.

### User Impact:
Keyboard only users while tabbing through the password textbox will not know that it is a required field.

**Comment by aasimkhan30** [comment]
The star is missing for all the fields and not just the password

**Comment by prabhudevu** [comment]
I have verified the issue in the below build

Version: 1.43.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/afafee844ce2b07eb55bdb6f40ee1ce88c0bbf4e
Date: 2023-03-29T05:28:54.263Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621

Issue is fixed.

**GithubTags:**#Closed;#Regressed:03-29-23;

![15766_Fixed](https://user-images.githubusercontent.com/98821792/228525263-0d0effee-3185-4819-9aac-e931833175cc.png)
